### ISSUE TITLE
Training Sample Data Does Not Return Valid Inference Results

### ISSUE THREAD

**Issue by sjgiewont** [andrewyguo/dope_training#4]
Hello I am trying to follow the instructions from the Isaac ROS Pose Estimation tutorial for [Training your own DOPE model](https://nvidia-isaac-ros.github.io/concepts/pose_estimation/dope/tutorial_custom_model.html#training-your-own-dope-model). I am trying to use the data found in [sample_data/](https://github.com/andrewyguo/dope_training/tree/master/sample_data), yet when I run inference on the generated model I do not get valid results.

To train I run the following:

`python -m torch.distributed.launch --nproc_per_node=1 train.py --data ./sample_data/ --object 003_cracker_box --epochs 100 --batchsize 24 --save`

Here is a screenshot of the loss curve i see in Tensorboard:

![image](https://github.com/andrewyguo/dope_training/assets/22220540/37dbe2b5-8c3c-4366-ae02-e77318ab3aeb)

To run inference i run the following. Note I am even running inference on the data used to train so I'd assume almost perfect results:

` python inference.py --weights ../output/weights/ --data ../sample_data/ --object 003_cracker_box`

The highest epoch that was actually able to return results was epoch 35 and go the following results:

![image](https://github.com/andrewyguo/dope_training/assets/22220540/9d88d1fe-6802-4137-8c7a-e7166b1da0bc)

Any thoughts or ideas. I've done many different experiments with varying epoch sizes, learning rates, etc. but have yet to see any decent results.

**Comment by andrewyguo** [comment]
Hello @sjgiewont , sorry for the late response. 

We are almost done moving this code base to the official repository at https://github.com/NVlabs/Deep_Object_Pose

There are still some bugs being ironed out right now. I will let you know once the code has been officially moved. if you are still seeing this issue on that new code base, please make an issue.

**Comment by andrewyguo** [comment]
Hello @sjgiewont 

The code base has officially been migrated to the official repository (https://github.com/NVlabs/Deep_Object_Pose)
